dev/io.openliberty.netty.internal.impl/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2025 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.1.0"
+                   xmlns:ibm="http://www.ibm.com/xmlns/appservers/osgi/metatype/v1.0.0">
+
+   <!-- NETTY OPTIONS ........................................................ -->
+
+   <OCD id="io.openliberty.netty.internal" ibm:alias="netty" name="internal" description="internal use only">
+      
+         <AD name="internal" description="internal use only"
```

Comment:
Just a general note: before we were to make any of this public, we would want to socialize it and include in the UFO for the Netty feature.